empire.go

```diff
@@ -323,8 +323,8 @@ func (e *Empire) ProcessesRun(ctx context.Context, app *App, opts ProcessRunOpts
 }
 
 // ReleasesFindByApp returns all Releases for a given App.
-func (e *Empire) ReleasesFindByApp(app *App) ([]*Release, error) {
-	return e.store.Releases(ReleasesQuery{App: app})
+func (e *Empire) ReleasesFindByApp(app *App, limit *int) ([]*Release, error) {
```

Comment:
I wanted to do this previously, but can we just replace this method with a:

```go
ReleasesFirst(q ReleasesQuery) ([]*Release, error)
```

And consume that from the http api.